packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageHeader.tsx

```diff
@@ -58,6 +59,9 @@ type TracePageHeaderEmbedProps = {
   showViewOptions: boolean;
   slimView: boolean;
   textFilter: string | TNil;
+  availableTagKeys: string[];
+  selectedMarkerColorKey: string[];
+  setMarkerColorKey: (tagKey: string[]) => void;
```

Comment:
These don't seem relevant in any way to the "page header" type. Is it possible to use the Context API to keep the knowledge of the settings only to the components that actually need this knowledge, such as the Settings popup and the SpanBar that renders the ticks?